src/lib/shell/streamer.cpp

```diff
@@ -87,5 +88,19 @@ void streamer_print_hex(streamer_t * self, const uint8_t * bytes, int len)
     }
 }
 
+void streamer_print_result(streamer_t * self, CHIP_ERROR error)
+{
+    VerifyOrReturn(error != CHIP_ERROR_IN_PROGRESS);
```

Comment:
Why this check?  This did not use to happen....

Reply:
Please document why it's here and how it's meant to be used, then it's probably fine.